vcpkg integrate bug

**Describe the bug**
I set the project’s use vcpkg to false
However, the installed directory of vcpkg is still in the includes

**Environment**
- OS: windows
- Compiler: msvc

**To Reproduce**
Steps to reproduce the behavior:
1. ./vcpkg integrate install

**Expected behavior**
![image](https://user-images.githubusercontent.com/68489543/205891312-fd2b742c-c555-4d55-9cf7-615ec23d8165.png)
![image](https://user-images.githubusercontent.com/68489543/205891325-816a577e-f1b0-42de-afd7-3704ca7b335b.png)

@chriscp-cat Thanks for posting this issue, 
1. The `Use vcpkg` function is set to `No`: it will disable the project to use the library and header files provided by `vcpkg`.

2. For `External Include Directories`, you need to run `./vcpkg integrate remove` and restart `VS`.

3. If you do not restart `vs`, the path of `vcpkg` still exists, but you cannot use `vcpkg`(`error LNK2019`). This is because the integration of `vcpkg` and vs (MSBuild) is implemented by importing`.props` and `.targets` files. Therefore, you need to run the command to restart VS and it will disappear.
View https://github.com/microsoft/vcpkg/blob/master/docs/users/buildsystems/msbuild-integration.md for more help.

--- a/scripts/buildsystems/msbuild/vcpkg.targets
+++ b/scripts/buildsystems/msbuild/vcpkg.targets
@@ -71,7 +71,7 @@
 
     <_ZVcpkgConfigSubdir Condition="'$(_ZVcpkgNormalizedConfiguration)' == 'Debug'">debug\</_ZVcpkgConfigSubdir>
     <_ZVcpkgExecutable>$(_ZVcpkgRoot)vcpkg.exe</_ZVcpkgExecutable>
-    <ExternalIncludePath>$(ExternalIncludePath);$(_ZVcpkgCurrentInstalledDir)include</ExternalIncludePath>
+    <ExternalIncludePath Condition="'$(VcpkgEnabled)' == 'true'">$(ExternalIncludePath);$(_ZVcpkgCurrentInstalledDir)include</ExternalIncludePath>
   </PropertyGroup>
 
   <PropertyGroup>

